index.bs

```diff
@@ -805,15 +805,15 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
 
     Constraints on pan influence camera selection through <a>fitness distance</a> toward cameras with the ability to pan. To exert this influence without overwriting the current pan setting, pan may be constrained to true. Conversely, constraining it to false disfavors cameras with the ability to pan.
 
-    Any algorithm which uses a {{MediaTrackConstraintSet}} object whose {{MediaTrackConstraintSet/pan}} dictionary member exists with a value other than false MUST either <a>request permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true, and, optionally, consider its {{DevicePermissionDescriptor/deviceId}} member set to any appropriate device's deviceId, or decide not to expose the pan setting.
+    Any algorithm which uses a {{MediaTrackConstraintSet}} object whose {{MediaTrackConstraintSet/pan}} dictionary member exists with a value other than false MUST either <a>request permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true, and, optionally, consider its {{DevicePermissionDescriptor/deviceId}} member set to any appropriate device's deviceId, and check that the permission is granted or decide not to expose the pan setting in <a>settings dictionaries</a> or in {{MediaTrackSettings}} dictionaries.
```

Comment:
This seems to say that the only penalty for not having permission is that `pan` will not be exposed in `getSettings`, which we already have superior language for [here](https://w3c.github.io/mediacapture-image/#dom-mediatracksettings-pan).

The sentence is also running long, making the and/or logic hard to follow. 

Instead, how about:  "If permission is not granted, the algorithm MUST ignore the member".

Reply:
...in order to burn out the ability to constrain on it.

Reply:
Done.